### ISSUE TITLE
Solved

### ISSUE THREAD

**Issue by muses0229** [OpenTexture/Paint3D#19]

**Comment by Zzlongjuanfeng** [comment]
yeah, we have discussed the multi-face problem in our [paper](https://arxiv.org/pdf/2312.13913). This issue primarily arises from the inconsistency of multi-view texture images sampled from the pre-trained 2D diffusion model, as it is not explicitly trained on multi-view datasets. 

Some new models ([zero123](https://openaccess.thecvf.com/content/ICCV2023/papers/Liu_Zero-1-to-3_Zero-shot_One_Image_to_3D_Object_ICCV_2023_paper.pdf), [EscherNet](https://arxiv.org/pdf/2402.03908)) consider camera pose during the diffusion process, which may alleviate the multi-face problem.
